src/Actions/CompileSvgsAction.php

```diff
@@ -30,6 +30,7 @@ public function execute(): void
             /** @var string $svgContent */
             $svgContent = file_get_contents($svg->getPathname());
             $svgContent = str_replace('<svg ', '<svg fill="currentColor" ', $svgContent);
+            $svgContent = str_replace('height="1em" ', ' ', $svgContent);
```

Comment:
Is this not removing a standardised height? 🤔 I'm assuming this change will just make the icons scale up.

Reply:
when you select an icon and copy the svg from its modal, it copies you with the height, since sometimes it's easier to just copy the icons you want than to download the entire suite

![image](https://github.com/owenvoke/blade-fontawesome/assets/4119035/16b884a3-4a34-4424-a4d7-3e8b69172cb2)